docs/docs-content/enterprise-version/upgrade/upgrade-k8s/airgap.md

```diff
@@ -0,0 +1,432 @@
+    ```
+
+    :::
+
+    </TabItem>
+
+    <TabItem value="aws-ecr" label="AWS ECR">
+
+    To get the authentication commands for your specific registries, navigate to the AWS ECR console and click **View
+    push commands**. Refer to the
+    [AWS ECR Authentication](https://docs.aws.amazon.com/AmazonECR/latest/userguide/getting-started-cli.html#cli-authenticate-registry)
+    docs for more details.
+
+    Alternatively, you can use the following command templates:
+
+    - Authenticate with your private `spectro-packs` registry.
+
+      ```shell
+      aws ecr get-login-password --region xxxxx | oras login --username AWS --password-stdin 1234567890.dkr.ecr.us-east-1.amazonaws.com
+      ```
+
+    - Authenticate with your public `spectro-images` registry.
+
+      ```shell
+      aws ecr-public get-login-password --region xxxxx | docker login --username AWS --password-stdin public.ecr.aws/xxxxxxx
+      ```
+
+    </TabItem>
+
+    </Tabs>
+
+3.  Prepare and export the environment variables required by the Palette airgap binary. Select one of the following tabs
+    for instructions on how to prepare these variables for Harbor and AWS ECR.
+
+    <Tabs>
+
+    <TabItem value="harbor" label="Harbor" default>
+
+    - `OCI_IMAGE_REGISTRY`: The IP address or domain name of the `spectro-images` registry.
+    - `OCI_IMAGE_BASE`: The namespace or repository that hosts the images.
+    - `OCI_PACK_REGISTRY`: The IP address or domain name of the `spectro-packs` registry.
+    - `OCI_PACK_BASE`: The namespace or repository that hosts the packages.
+
+      ```shell
+      export OCI_IMAGE_REGISTRY=<harbor-endpoint>
+      export OCI_IMAGE_BASE=spectro-images
+      export OCI_PACK_REGISTRY=<harbor-endpoint>
+      export OCI_PACK_BASE=spectro-packs
+      ```
+
+    Consider the following example for reference.
+
+    ```shell
+    export OCI_IMAGE_REGISTRY=example.internal.com
+    export OCI_IMAGE_BASE=spectro-images
+    export OCI_PACK_REGISTRY=10.10.100.48
+    export OCI_PACK_BASE=spectro-packs
+    ```
+
+    </TabItem>
+
+    <TabItem value="aws-ecr" label="AWS ECR">
+
+    - `ECR_IMAGE_REGISTRY`: The IP address or domain name of the public `spectro-images` registry.
+    - `ECR_IMAGE_BASE`: The namespace or repository that hosts the images.
+    - `ECR_IMAGE_REGISTRY_REGION`: The AWS region where the ECR registry is located.
+    - `ECR_PACK_REGISTRY`: The IP address or domain name of the public `spectro-packs` registry.
+    - `ECR_PACK_BASE`: The namespace or repository that hosts the packages.
+    - `ECR_PACK_REGISTRY_REGION`: The AWS region where the ECR registry is located.
+
+      ```shell
+      export ECR_IMAGE_REGISTRY=<ecr-endpoint>
+      export ECR_IMAGE_BASE=spectro-images
+      export ECR_IMAGE_REGISTRY_REGION=<ecr-region>
+      export ECR_PACK_REGISTRY=<ecr-endpoint>
+      export ECR_PACK_BASE=spectro-packs
+      export ECR_PACK_REGISTRY_REGION=<ecr-region>
+      ```
+
```

Comment:
I think we need a `<br /` or maybe a prettier ignore to fix the indentation.

![CleanShot 2024-04-08 at 08 18 21](https://github.com/spectrocloud/librarium/assets/29551334/4c514301-9483-418b-8315-64efcda2824b)